README.markdown

```diff
@@ -89,6 +89,56 @@ docker-compose exec backend bash
 You can follow [coffeescript-styleguide](https://github.com/koding/styleguide-coffeescript)
 that we are relying on.
 
+## Run Koding on Local Machine
+
+If you wish to work on Koding itself, you need to install following software packages:
+
+### Software Prerequisites
+
+- [Golang](http://www.golang.org/) v1.7
+- [Node.js](https://nodejs.org/en/) v0.10
+- [Coffeescript](http://coffeescript.org/)
+- [Supervisord](http://supervisord.org/)
+
+### Start developing
+
+If you have the above software packages installed on your computer, you can
+follow steps for running the instance:
+
+```bash
+git clone https://github.com/koding/koding.git /your/koding/path
+cd /your/koding/path
+node -v # make sure your node version is not greater than `0.10.x`
+coffee -v # make sure this doesn't return an error
+npm install
+```
+
+You should have packages ready for running build specific scripts.
+
+```bash
+cd /your/koding/path
```

Comment:
Redundant because user is already changed to koding clone path in the previous snippet. You might as well merge snippets.